napari/_qt/qt_viewer.py

```diff
@@ -540,15 +540,16 @@ def console_backlog(self):
         return self._console_backlog
 
     def _get_console(self) -> Optional[QtConsole]:
-        """
-        Function for setup console.
+        """Function to setup console.
 
         Returns
         -------
+        console : Optional[QtConsole]
```

Comment:
in other docstrings in this file, we have "type or None"

So maybe, for consistency:
```suggestion
        console : QtConsole or None
```

We should decide how to format docstrings

Reply:
> We should decide how to format docstrings

FYI there's nothing necessary to decide, the convention here is to use numpy-style docstrings.